BE/src/main/java/louie/dong/airbnb/book/Book.java

```diff
@@ -18,28 +18,30 @@
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class Book {
 
-	@Id @GeneratedValue(strategy = GenerationType.IDENTITY)
-	private Long id;
-	@ManyToOne(fetch = FetchType.LAZY)
-	@JoinColumn
-	private Accommodation accommodation;
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn
+    private Accommodation accommodation;
 
-	private LocalDateTime checkIn;
-	private LocalDateTime checkOut;
-	private int guestCount;
-	private int finalPrice;
-	private boolean isCanceled;
+    private LocalDateTime checkIn;
+    private LocalDateTime checkOut;
+    private int guestCount;
+    private int finalPrice;
+    private boolean isCanceled;
```

Comment:
```suggestion
    private boolean isCanceled = false;
```

필드에도 기본값 선언이 가능합니다 :)

Reply:
생성자가 많아진다면 필드에 기본값을 선언해주는게 훨씬 좋겠네요 😀
참고하겠습니다!